### PR TITLE
Doc linux only supports x86-64

### PR DESCRIPTION
**Pull request by jkonrath-postman**

### CODE REVIEW

File: src/pages/docs/getting-started/installation-and-updates.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ contextual_links:
 
 ---
 
-Postman is available on the web at **[go.postman.co/home](https://go.postman.co/home)** and as a native desktop app for Mac (Intel and Apple), Windows (32-bit / 64-bit), and Linux (64-bit) operating systems.
+Postman is available on the web at **[go.postman.co/home](https://go.postman.co/home)** and as a native desktop app for Mac (Intel and Apple), Windows (Intel 32-bit / 64-bit), and Linux (64-bit) operating systems.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/installation-and-updates.md:36]:
```suggestion
Postman is available on the web at **[go.postman.co/home](https://go.postman.co/home)** and as a native desktop app for Mac (Intel or Apple silicon), Windows (32-bit or 64-bit), and Linux (64-bit).
```

Comment (mark-anthony-postman) [reply src/pages/docs/getting-started/installation-and-updates.md:36]: I wouldn't specify "Intel" for Windows. For example, I'm sure AMD chips work fine. I'd leave it as (32-bit / 64-bit).

File: src/pages/docs/getting-started/installation-and-updates.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ Postman is available for Windows 7 and later.
 1. [Download](https://www.postman.com/downloads/) the latest Postman version.
 1. Select and run the `.exe` file to install Postman.
 
-> Postman v9.4 is the last version of Postman that supports both 32-bit and 64-bit Windows. All versions of Postman following v9.4 will only work on 64-bit Windows. You can continue to use Postman v9.4 and earlier on 32-bit Windows.
+> Postman v9.4 is the last version of Postman that supports both 32-bit and 64-bit Windows. All versions of Postman following v9.4 will only work on 64-bit x86 Windows. You can continue to use Postman v9.4 and earlier on 32-bit Windows.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/installation-and-updates.md:65]: Are we saying Postman won't run (in emulation mode) on Windows for ARM processors? (Not that there are a lot Windows systems running ARM out there, but there are some)

**Review by mark-anthony-postman — APPROVED** [review]
Left a few questions/comments. Otherwise LGTM

File: src/pages/docs/getting-started/installation-and-updates.md (hunk 2 above)
Comment (jkonrath-postman) [reply src/pages/docs/getting-started/installation-and-updates.md:65]: Correct. (There was an internal thread about this in Slack, mostly about trying to run Windows ARM in Parallels on an M1.)

File: src/pages/docs/getting-started/installation-and-updates.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/getting-started/installation-and-updates.md:36]: Okay then I would say "x86" instead of "Intel" for Windows

**Pull request merged**